R/compat-dplyr.R

```diff
@@ -1,106 +1,62 @@
 #' Compatibility with dplyr
 #'
 #' @description
-#' rsample should be fully compatible with dplyr 1.0.0.
+#' This page lays out the compatibility between rsample and dplyr. The `rset`
+#' objects from rsample are a sepeciifc subclass of tibbles, hence standard
+#' dyplyr operations like joins as well row or column modifications work.
```

Comment:
```suggestion
#' objects from rsample are a specific subclass of tibbles, hence standard
#' dplyr operations like joins as well row or column modifications work.
```

Maybe do one more full pass for typos?

Reply:
thanks!